custom/icds_reports/management/commands/sql_scripts/ap_data_pull_2.sql

```diff
@@ -0,0 +1,18 @@
+
+COPY(SELECT
+    awc.state_name,
+    awc.district_name,
+    awc.block_name,
+    awc.supervisor_name,
+    awc.awc_name,
+    awc.awc_site_code,
+    CASE WHEN (agg.vhnd_conducted IS NOT NULL AND agg.vhnd_conducted>0) THEN 'Y'
```

Comment:
@Rohit25negi Is this correct?

Reply:
looks correct. Can you use `num_awcs_conducted_vhnd` column instead? I doubt vhnd_conducted goes that back in time.